treeherder/etl/resultset_loader.py

```diff
@@ -45,6 +45,8 @@ def get_transformer_class(self, exchange):
                 return GithubPushTransformer
             elif exchange.endswith("pull-request"):
                 return GithubPullRequestTransformer
+        elif "hgpushes" in exchange:
```

Comment:
I know this is mirroring the logic for GitHub pulse messages, but I wonder if this string should be more specific? eg include forward slashes to prevent it matching on something else?